products/samsungmobile.md

```diff
@@ -665,78 +668,78 @@ releases:
 -   releaseCycle: "galaxy-f02s"
     releaseLabel: "Galaxy F02s"
     releaseDate: 2021-04-09
-    eoas: false
-    eol: false
+    eoas: 2023-04-09
+    eol: 2024-04-09
     link: https://doc.samsungmobile.com/SM-E025F/INS/doc.html
 
 -   releaseCycle: "galaxy-a52"
     releaseLabel: "Galaxy A52"
     releaseDate: 2021-03-26
-    eoas: false
-    eol: false
+    eoas: 2024-03-26
+    eol: 2025-03-26
     link: https://doc.samsungmobile.com/SM-A525F/XID/doc.html
 
 -   releaseCycle: "galaxy-a72"
     releaseLabel: "Galaxy A72"
     releaseDate: 2021-03-26
-    eoas: false
-    eol: false
+    eoas: 2024-03-26
+    eol: 2025-03-26
     link: https://doc.samsungmobile.com/SM-A725F/XEO/doc.html
 
 -   releaseCycle: "galaxy-m12-india"
     releaseLabel: "Galaxy M12 (India)"
     releaseDate: 2021-03-18
-    eoas: true
+    eoas: 2024-11-05
     eol: 2024-11-05
     link: https://doc.samsungmobile.com/SM-M127G/INS/doc.html
 
 -   releaseCycle: "galaxy-a52-5g"
     releaseLabel: "Galaxy A52 5G"
     releaseDate: 2021-03-17
-    eoas: false
-    eol: false
+    eoas: 2024-03-17
+    eol: 2025-03-17
     link: https://doc.samsungmobile.com/sm-a526b/dbt/doc.html
 
 -   releaseCycle: "galaxy-xcover-5"
     releaseLabel: "Galaxy Xcover 5"
     releaseDate: 2021-03-12
-    eoas: false
-    eol: false
+    eoas: 2024-03-12
+    eol: 2025-03-12
     link: https://doc.samsungmobile.com/SM-G525F/XNZ/doc.html
 
 -   releaseCycle: "galaxy-m62"
     releaseLabel: "Galaxy M62"
     releaseDate: 2021-03-03
-    eoas: false
-    eol: false
+    eoas: true
+    eol: true
```

Comment:
```suggestion
    eol: 2024-03-03
```

Conservative choice, not listed on https://security.samsungmobile.com/workScope.smsb.